yarn.lock

```diff
@@ -1488,6 +1488,41 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@carbon/react@npm:^1.12.0":
+  version: 1.55.0
```

Comment:
Did you intend to do a Carbon bump?

Reply:
This was a side effect of the patient commons lib being added. It requires ^1.55.0

Reply:
The commons lib requires [v^1.12.0](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-common-lib/package.json#L32) but currently hydrated with `v1.40.0`